src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/Swagger/SwaggerEnumSchemaFilter.cs

```diff
@@ -23,6 +24,44 @@ public void Apply(OpenApiSchema schema, SchemaFilterContext context)
                 $"Must use an enum type for '{nameof(SwaggerEnumAttribute)}'");
         }
 
+        var schemaType = schema.Type == "array" ? schema.Items.Type : schema.Type;
```

Comment:
I'm going to assume that you know exactly what is happening here :)

Reply:
Yep, in where we have an array of items, we're getting the underlying `Type` for each array item. Or, if we don't have a list, we just use the schema `Type` as-is.